.github/workflows/flaky-test-monitor.yml

```diff
@@ -1,14 +1,20 @@
-name: Flaky Test Summary
+name: Flaky Test Monitor
 
 on:
+  schedule:
+    - cron: '0 */4 * * *' # every 4 hours
   push:
     branches:
       - master
+  pull_request: # TODO: remove this before merging
```

Comment:
why do we run on every PR?

Reply:
As the comment says, I'm gonna remove this before merging.

It's just there now so that every time I push, the jobs are triggered on this PR. It provides a convenient way to test my changes.